charmcraft/commands/build.py

```diff
@@ -206,6 +202,11 @@ def build_charm(self, bases_config: BasesConfiguration) -> str:
 
         self._handle_deprecated_cli_arguments()
 
+        if is_charmcraft_running_in_managed_mode():
+            work_dir = env.get_managed_environment_home_path()
```

Comment:
Maybe the `logger.debug` call five lines above should be fixed?

Reply:
Indeed. Will fix to display `work_dir` instead.